src/main/java/sirius/web/data/ExcelExport.java

```diff
@@ -65,8 +68,8 @@
 
     private final Workbook workbook;
     private Sheet currentSheet;
-    private int rows = 0;
-    private int maxCols = 0;
+    private final Map<String, AtomicInteger> rows = new HashMap<>();
```

Comment:
shoudln't we use the sheet index here, as the name can be non-unique(?)

Reply:
Names are unique, but pushing a version based on the index.